doc/tutorial.md

```diff
@@ -0,0 +1,175 @@
+# Getting Started with Submitting Job
+
+## Download and Configure paddlectl
+
+`paddlectl` is a command-line tool that submits distributed training job to the cloud.
+
+- Step1
+
+  Download `paddlectl` and copy the binary file to the folder which under
+  environment variable `$PATH`, such as `/usr/local/bin`, then set the
+  executable access by command `chmod +x /usr/local/bin/paddlectl`.
+
+  We advise users to download the last tag `paddlectl` on
+  the [Release Page](https://github.com/PaddlePaddle/cloud/releases),
+  and you can also fetch the least version binary file from our CI system,
+  it's built by the last code on develop branch.
+
+  OS |  Link
+  -- | --
+  Mac OSX| [paddlecloud.darwin](http://guest:@paddleci.ngrok.io/repository/download/PaddleCloud_Client/.lastSuccessful/paddlecloud.darwin)
+  Windows| [paddlecloud.exe](http://guest:@paddleci.ngrok.io/repository/download/PaddleCloud_Client/.lastSuccessful/paddlecloud.exe)
+  Linux | [paddlecloud.x86_64](http://guest:@paddleci.ngrok.io/repository/download/PaddleCloud_Client/.lastSuccessful/paddlecloud.x86_64)
+
+- Step2
+
+  Configurate the configuration file `~/.paddle/config` (`./paddle\config`
+  under current user folder for Windows),
+  an example is as following:
+
+  ```bash
+  datacenters:
+  - name: production
+    username: paddlepaddle
+    password: paddlecloud
+    endpoint: http://production.paddlecloud.com
+  - name: experimentation
+    username: paddlepaddle
+    password: paddlecloud
+    endpoint: http://experimentation.paddlecloud.com
+  current-datacenter: production
+  ```
+
+  Supports you have two data-center's access, one for `production` and another one for
+  `experimentation`, you can choose one data-center by the `current-datacenter` field
```

Comment:
"you can choose one data-center by the `current-datacenter` field for the current one." => "you can select your current data-center by editing `current-datacenter` field"